proxy/src/proxy.rs

```diff
@@ -69,7 +69,8 @@ impl ProxyConnection {
             let (username, dbname) = self.handle_startup()?;
 
             // Both scenarios here should end up producing database credentials
-            if username.ends_with("@zenith") {
+            // HACK, will remove this before PR, just ignore it
+            if true || username.ends_with("@zenith") {
```

Comment:
Yeah, we want to remove that implicit behavior https://github.com/zenithdb/zenith/issues/920 and just setup one proxy for the new users (e.g. start.zenith.tech) and one for users with credentials (e.g. cloud.zenith.tech) -- so that thing will be a cli flag